src/features/click-to-play.js

```diff
@@ -1229,6 +1623,53 @@ function getLearnMoreLink (mode) {
     return linkElement
 }
 
+/**
+ * Reads and stores a set of styles from the original tracking element, and then returns it.
+ * @param {Element} originalElement Original tracking element (ie iframe)
+ * @param {DuckWidget} widget The widget Object.
+ * @returns {{[key: string]: string[]}} Object with styles read from original element.
+ */
+function getOriginalElementStyle (originalElement, widget) {
+    if (widget.originalElementStyle) {
+        return widget.originalElementStyle
+    }
+
+    const stylesToCopy = ['display', 'visibility', 'position', 'top', 'bottom', 'left', 'right',
+        'transform', 'margin', 'width', 'max-width', 'height', 'max-height']
+    widget.originalElementStyle = {}
+    const allOriginalElementStyles = getComputedStyle(originalElement)
+    for (const key of stylesToCopy) {
+        widget.originalElementStyle[key] = [
```

Comment:
Does this array have defined meaning rather than a set of the same data?

- If so, I would instead use an object with named keys. for example:
```
{
  value: allOriginalElementStyles[key],
  priority: originalElement.style.getPropertyPriority(key)
}
```

**However:** I see you indexing the 0th element, are you using the 1st element? Can we remove the priority key altogether? (this would simplify all the array access across the file for this key)